common/persistence/src/test/java/app/coronawarn/server/common/persistence/repository/DiagnosisKeyRepositoryTest.java

```diff
@@ -18,12 +19,12 @@
 @DataJdbcTest
 public class DiagnosisKeyRepositoryTest {
 
-  @Autowired
-  private DiagnosisKeyService service;
-
   @Autowired
   private DiagnosisKeyRepository repository;
 
+  private byte[] id = new byte[16];
```

Comment:
Maybe we can move this variable into the respective tests?

Reply:
It is used in both of the tests, that's why i thought to have it as a global variable